what-the-stack/src/main/java/com/haroldadmin/whatthestack/ui/components/IconOutlinedButton.kt

```diff
@@ -0,0 +1,38 @@
+package com.haroldadmin.whatthestack.ui.components
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+
+@Composable
+internal fun OutlinedIconButton(
```

Comment:
The file name is `IconOutlinedButton`, you should probably rename it to match the function.